scripts/ubuntu/build_openvslam_docker.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+TOPDIR=$PWD/../../ #openvslam directory
+
+( 
+    cd $TOPDIR
+
+    # Building Docker Image
+    docker build -t openvslam-desktop -f Dockerfile.desktop . --build-arg NUM_THREADS=7
```

Comment:
I think the user needs a high performance CPU in order to set NUM_THREADS to 7. I think lowering the default value of NUM_THREADS is more suitable for beginners.

Reply:
Sorry for my laziness. I updated it to use nproc which is more proper to specify the number.

Reply:
BTW I use "nproc - 1" because if I use full cpu, it might be freezed.